tests/checks/integration/test_iis.py

```diff
@@ -0,0 +1,77 @@
+# 3p
+from nose.plugins.attrib import attr
+
+# project
+from checks import AgentCheck
```

Comment:
Not needed, we can avoid using it by calling the `assertServiceCheckOK`, `assertServiceCheckCritical`, etc. Let's 🔥  it!